CRM/Core/Payment/PayPalImpl.php

```diff
@@ -793,23 +813,28 @@ public function doTransferCheckout(&$params, $component = 'contribute') {
       CRM_Core_Error::fatal(ts('Component is invalid'));
     }
 
-    $notifyURL = $config->userFrameworkResourceURL . "extern/ipn.php?reset=1&contactID={$params['contactID']}" . "&contributionID={$params['contributionID']}" . "&module={$component}";
+    $args = array(
```

Comment:
What do these args add? They contributionID & module are returned from paypal std & the processorID is part of the parent method - ie       'civicrm/payment/ipn/' . $this->_paymentProcessor['id'],